### ISSUE TITLE
Support image ALT text when using photos

### ISSUE THREAD

**Issue by gijswijs** [barryf/micropublish#101]
alt is a required attribute of the <img> element and should be provided in any use of images on indieweb pages from [photo](https://indieweb.org/photo) posts.

source: https://indieweb.org/alt

**Comment by barryf** [comment]
You're right. This is important for accessibility and should be supported.

I'll have a think about how to implement this because currently photos are specified as a whitespace-delimited list of URLs.
